docs/alternator/compatibility.md

```diff
@@ -108,38 +108,71 @@ exactly the same microsecond-resolution timestamp, the the result may be
-by providing an entry in Scylla configuration:
-    `alternator_enforce_authorization: true`
-
-Although Alternator implements DynamoDB's authentication, including the
-possibility of listing multiple allowed key pairs, there is currently no
-implementation of _access control_. All authenticated key pairs currently get
-full access to the entire database. This is in contrast with DynamoDB which
-supports fine-grain access controls via "IAM policies" - which give each
-authenticated user access to a different subset of the tables, different
-allowed operations, and even different permissions for individual items.
-All of this is not yet implemented in Alternator.
-See <https://github.com/scylladb/scylla/issues/5047>.
+Alternator also implements authorization, or _access control_ - defining
+which authenticated user is allowed to do which operation, such as reading
+or writing to a specific table. The way this is supported in Alternator is
+currently _not_ compatible with DynamoDB's APIs (IAM or PutResourcePolicy).
+Instead, one needs to grant permisions to specific roles using CQL, with
+the "GRANT" command. For example, an Alternator table "xyz" is visible to
+CQL as `alternator_xyz.xyz`, and the following command will allow requests
+from user "myrole" to read this table (with GetItem and other read operations):
+`GRANT SELECT ON alternator_xyz.xyz TO myrole`. Refer to the CQL documentation
+on how to use GRANT and REVOKE to control permissions, and which permissions
+exist:
+<https://docs.scylladb.com/stable/operating-scylla/security/authorization.html>
+
+The following permissions are needed to run the following API operations:
+
+ * SELECT:      GetItem, Query, Scan, BatchGetItem, GetRecords
+ * MODIFY:      PutItem, DeleteItem, UpdateItem, BatchWriteItem
+ * CREATE:      CreateTable
+ * DROP:        DeleteTable
+ * ALTER:       UpdateTable, TagResource, UntagResource, UpdateTimeToLive
+ * none needed: ListTables, DescribeTable, DescribeEndpoints,
+                ListTagsOfResource, DescribeTimeToLive,
+                DescribeContinuousBackups, ListStreams, DescribeStream,
+                GetShardIterator
+
+Note that the required permissions depend on the type of operation, not on
+what it does. For example, even though the PutItem operation can read the
+value of an item (when used with `ReturnValues=ALL_OLD`), it still requires
+the MODIFY permission, not the SELECT permission.
+
+Permissions are separate for a base table and each of its GSI/LSI and CDC
+log, so it's possible to give a role permissions to read one index and
+not the base, or vice versa, and so on. To build the GRANT command, you
+need to know the CQL name of each of these objects. For example, the CQL
+name of GSI "abc" of Alternator table "xyz" is `alternator_xyz.xyz:abc`.
+If you don't know the name of the table, you can try a forbidden operation
+and the AccessDeniedException error will contain the name of the table
+that was lacking permissions.
 
```

Comment:
Please add references in getting-started.md so people will know where to find authentication and authorization.  In fact it probably deserves its own page, but that can be done as a follow-up.

Reply:
Done (add a reference, not a separate page).
@tzach also thinks the compatibility.md page should be split - see https://github.com/scylladb/scylladb/issues/19822, but I think it shouldn't be split too much. Perhaps indeed the authentication/authorization part deserves a separate page.